modules/serverpod_auth/serverpod_auth_server/lib/src/generated/email_auth.dart

```diff
@@ -272,6 +281,7 @@ class EmailAuthRepository {
     );
   }
 
+  /// Inserts a single [EmailAuth] and returns the inserted row.
```

Comment:
here we should hint that only the returned entity will have the entity.

The same also probably applies for attach/detach.

Reply:
Agreed!